app/upgrades/1.0/upgrades.go

```diff
@@ -0,0 +1,16 @@
+package v10
+
+import (
+	"context"
+
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+)
+
+const Name = "0.5-to-1.0"
+
+func UpgradeHandler(mm *module.Manager, configurator module.Configurator) upgradetypes.UpgradeHandler {
```

Comment:
Needs to include contents of 0.5 upgrade handler. Alternatively, just rename what's in 0.5 to 1.0